app/Services/OrganizationService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Models\Organization;
+use App\Models\User;
+
+class OrganizationService
+{
+    public function create(array $organization): void
+    {
+        if ($organization["type"] === "Organization") {
```

Comment:
I would like to move `"Organization"` into some const.

Reply:
moved it to a const